lib/web/ui/app.go

```diff
@@ -155,6 +172,13 @@ func MakeApp(app types.Application, c MakeAppsConfig) App {
 	return resultApp
 }
 
+func mapPermissionSet(ps *types.IdentityCenterPermissionSet) IdentityCenterPermissionSet {
```

Comment:
Do we really need a separate function for type conversion if it is only used once in the code?
The map... naming is misaligning suggesting that there map... function does some kine of logic but this is just simple conversion.